src/components/CodeView/index.tsx

```diff
@@ -114,26 +124,16 @@ export class CodeViewBase extends React.Component<Props> {
     let slowAlert;
 
     if (
-      codeShouldBeTrimmed({
+      _codeShouldBeTrimmed({
```

Comment:
tbh I don't really understand why our(? React?) code is written like this - `codeShouldBeTrimmed` is a function we import... so why are we declaring it in `DefaultProps`, which then has to be set in `defaultProps` and again in `renderWithLinterInfo` - it's like 3 extra lines of code cruft for this function alone.

Reply:
That's dependency injection, mainly for testing purposes.